core/src/main/resources/error/delta-error-classes.json

```diff
@@ -662,5 +662,13 @@
   "RESERVED_CDC_COLUMNS_ON_WRITE" : {
     "message" : [ "", "The write contains reserved columns <columnList> that are used", "internally as metadata for Change Data Feed. To write to the table either rename/drop", "these columns or disable Change Data Feed on the table by setting", "<config> to false." ],
     "sqlState" : "42000"
+  },
+  "INTERLEAVING_COLUMN_DOES_NOT_EXIST" : {
```

Comment:
All Delta error classes should have `DELTA_` prefix. and we should be consistent in name and message to use Zorder*** and not "interleaving" which is internal details